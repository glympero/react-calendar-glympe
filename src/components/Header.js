import React from 'react';
import { NavLink } from 'react-router-dom';
const Header = () => (
    <header>
      <h1>Code Exercise for JS/React Developer</h1>
      <div className="container">
        <NavLink to="/" className="item" activeClassName="is-active" exact={true}>Calendar</NavLink>
        <NavLink to="/about" className="item" activeClassName="is-active">Help Page</NavLink>
      </div>
    </header>
  );

export default Header;