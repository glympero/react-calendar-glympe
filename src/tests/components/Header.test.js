
import { shallow } from 'enzyme';
import React from 'react';

import Header from '../../components/Header';

test('should render header correctly', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper.find('h1').length).toBe(1);
    expect(wrapper.find('h1').text()).toBe('Code Exercise for JS/React Developer');

    expect(wrapper).toMatchSnapshot();
});