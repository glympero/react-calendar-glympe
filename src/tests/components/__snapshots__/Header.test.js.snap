// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Code Exercise for JS/React Developer
  </h1>
  <div
    className="container"
  >
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      className="item"
      exact={true}
      to="/"
    >
      Calendar
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      className="item"
      to="/about"
    >
      Help Page
    </NavLink>
  </div>
</header>
`;
